waisntechtools/alerts/recent_twilio_messages.py

```diff
@@ -0,0 +1,22 @@
+from django.conf import settings
+from django.http import HttpResponseForbidden
+from twilio.rest import Client
+from twilio.request_validator import RequestValidator
+
+import requests
+import datetime
+
+
+def recent_twilio_messages():
+
+    if settings.DEBUG == True:
+        request_valid = True
+
+    if request_valid:
+        client = Client(settings.TWILIO_ACCOUNT_SID,
+                        settings.TWILIO_AUTH_TOKEN)
```

Comment:
The client is being instantiated the same way multiple times, which means that it should be abstracted out.